NumberBaseball/NumberBaseball/main.swift

```diff
@@ -6,5 +6,53 @@
 
 import Foundation
 
-print("Hello, World!")
+typealias resultType = (strike: Int, ball: Int)
+
+func randomNumbers() -> Array<Int> {
+    var computerNumbers: Array<Int> = []
+    
+    while computerNumbers.count < 3 {
+        if computerNumbers.count == 3 {
+            break
+        }
+        let randomNumber = Int.random(in:1...9)
+        if computerNumbers.contains(randomNumber) {
+            continue
+        }
+        computerNumbers.append(randomNumber)
+    }
+    return computerNumbers
+}
+
+func compareNumbers(user userNumbers: Array<Int>, to computerNumbers: Array<Int>) -> resultType {
+    let strikes = userNumbers.enumerated().filter {userNumbers[$0.offset] == computerNumbers[$0.offset]}.map {$0.element}
```

Comment:
enumeraterd와 고차함수를 사용하여 strike의 개수를 찾으신 것은 정말 좋으신 거 같습니다. 
하지만 가독성이 좀 떨어져 보이네요.  고차함수를 연결할 때 아래와 같이 중괄호 사이를 띄는 컨벤션을 사용하는 편입니다. 
```suggestion
    let strikes = userNumbers.enumerated().filter { userNumbers[$0.offset] == computerNumbers[$0.offset] }.map { $0.element }
```
현재 코드가 100자를 넘어가는 것으로 보이는데 보통 100자를 넘기지 않게 코드를 작성하는 편입니다. 개행을 하거나 변수를 주어 코드수를 조금 줄여보시는 것이 어떨까요? 
[Style Guide](https://google.github.io/swift/)를 참고하시면 좋을거 같네요.
(팁! Xcode -> 설정 -> Text Editing -> Page guide at column 에서 가이드라인을 설정할 수 있어요!